pilot/pkg/xds/eds.go

```diff
@@ -139,7 +141,8 @@ func (s *DiscoveryServer) EDSCacheUpdate(shard model.ShardKey, serviceName strin
 // It also tracks the changes to ServiceAccounts. It returns whether endpoints need to be pushed and
 // it also returns if they need to be pushed whether a full push is needed or incremental push is sufficient.
 func (s *DiscoveryServer) edsCacheUpdate(shard model.ShardKey, hostname string, namespace string,
-	istioEndpoints []*model.IstioEndpoint) PushType {
+	istioEndpoints []*model.IstioEndpoint,
+) PushType {
```

Comment:
suggest stop changing unneeded format within bug fix

Reply:
This is unintentional because of automated gofmt (IT has this new format). I will revert.